staging/src/k8s.io/client-go/go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f // indirect
```

Comment:
TODO @aojea , see if we can move to the std library and remove this dependency https://pkg.go.dev/golang.org/x/time/rate